packages/marketprice/src/alarms/mod.rs

```diff
@@ -395,6 +395,68 @@ pub mod tests {
         assert_eq!(None, triggered_alarms.next());
     }
 
+    #[test]
+    fn test_out_for_delivery_removes_above() {
+        const PRICE_BASE: Amount = 1;
+        const PRICE_QUOTE: Amount = if let Some(quote) = PRICE_BASE.checked_add(1) {
+            quote
+        } else {
+            panic!()
+        };
```

Comment:
[everywhere] It's an overkill ... please use the operator + and - ... after all we always panic when overflow